lib/player/stream_video_source.js

```diff
@@ -1110,7 +1110,17 @@ shaka.player.StreamVideoSource.prototype.startStreams_ = function(
   // Set the video's current time before starting the streams so that the
   // streams begin buffering at the stream start time.
   shaka.log.info('Starting each stream from', streamStartTime);
-  this.video.currentTime = streamStartTime;
+
+  function setInitialCurrentTime() {
```

Comment:
If this is run in the context of video, please annotate with @this {!HTMLVideoElement}.  Alternately, use bind and run it in the context of StreamVideoSource.